tasks.sh

```
@@ -152,37 +166,40 @@ while [[ $# > 0 ]]; do
         export BUILD_TARGET="$2"
         if [[ $BUILD_TARGET == "demos" ]]; then
-            export CMAKE_CONFIG_DEFINE="$CMAKE_CONFIG_DEFINE -DEGE_BUILD_DEMO=ON"
+            CMAKE_CONFIG_DEFINE+=("-DEGE_BUILD_DEMO=ON")
         fi
         shift
         shift
         ;;
     --toolset)
         echo "set toolset to $2"
-        export CMAKE_CONFIG_DEFINE="$CMAKE_CONFIG_DEFINE -T $2"
+        CMAKE_CONFIG_DEFINE+=(-T $2)
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)